gradle/bintray.gradle

```diff
@@ -1,5 +1,5 @@
 apply plugin: 'com.jfrog.bintray'
-apply plugin: 'com.github.dcendents.android-maven'
+//apply plugin: 'com.github.dcendents.android-maven'
```

Comment:
this plugin clashes with the `maven-push.gradle` from what I gather it is due to both trying to bring in an install task.  I can't seem to find why `com.github.dcendents.android-maven` is needed unless bintray deployment depends on it.